appserver/pom.xml

```diff
@@ -96,7 +96,7 @@
         <jakarta.interceptor-api.version>2.1.0</jakarta.interceptor-api.version>
 
         <!-- Jakarta Security + Authentication/Authorization -->
-        <soteria.version>3.0.3</soteria.version>
```

Comment:
Is it a mistake or some experiment?

Reply:
My mistake to publish that. I wanted to build locally w/o artifacts from staging.